synapseclient/core/upload/multipart_upload.py

```diff
@@ -531,9 +530,14 @@ def multipart_upload_string(
      https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.17
     """
 
+    def md5_fn(part, _):
```

Comment:
Should this be a utils function?

Reply:
Moved over to a utils function: https://github.com/Sage-Bionetworks/synapsePythonClient/pull/999/commits/4cd8886d95f2d23dc78406589aa1e83b3156ed2f